src/main/java/gregtech/api/util/GTUtility.java

```diff
@@ -1114,6 +1114,21 @@ public static int tryParseInt(String val, int defaultValue){
         return defaultValue;
     }
 
+    /**
+     * Tries to parse a string into an int, returning a default value if it fails.
```

Comment:
long, not an int

Reply:
resolved with https://github.com/GregTechCEu/GregTech/pull/1511/commits/b497de7a6d413153cc688789fbe90260f9702850